docs/UsersGuide/best/ground-interface.md

```diff
@@ -0,0 +1,193 @@
+# Ground Interface Architecture and Customization
+
+This guide will discuss the F´ ground interface layers and how to customize them. There are two parts to the ground
+interface: the spacecraft side, and the ground side. This guide will primarily focus on the spacecraft side adaptation
+as the most common pattern is to adapt F´ flight software for some other ground system (Cosmos, OpenMCT, etc). This
```

Comment:
Link to COSMOS: https://github.com/BallAerospace/COSMOS

Reply:
Since they are both available publicly, I suppose I can link to them.  I don't want to give the impression that we support that "today", but links would be nice.